pkg/scripts/cleanup.sh

```diff
@@ -7,11 +7,13 @@
 # -h: help
 set -e
 CURRENT_PATH="$(pwd)"
-SCRIPT_PATH="$(dirname "$0")"
-ROOT_PATH="${SCRIPT_PATH}/../.."
-cd "${ROOT_PATH}"
-TAG_MODULE_PATH="./pkg/modules/AWS/asg_ec2_tagging"
-TFSTATE_PATH="./pkg/tfstate"
+# this script is landed in {repo_root_path}/pkg/scripts to to get root path we should back two level from script path
+cd "$(dirname "$0")/../.."
```

Comment:
Do we need to still switch directories if we use absolute paths?

Reply:
No, this switching directory is just to get the clean path without back and forth between script folder and root path of repo.